infra/modules/gcp-oauth-secrets-role/main.tf

```diff
@@ -0,0 +1,14 @@
+resource "google_project_iam_custom_role" "secret_manager_updater" {
+  project     = var.project_id
+  role_id     = "SecretManagerAccess"
```

Comment:
role id probably needs env/namespace component, to avoid collisions across deployments. (some customers mix dev+prod stuff in same host account/project, distinguishing with env prefixes)

Reply:
maybe call it `PsoxyInstanceSecretManager` or something?

Reply:
or `PsoxyInstanceSecretLocker`? 

labels just being implementation detail of how we lock/unlock a secret.

Reply:
Ok, moved to GCP module